users/api/requests.go

```diff
@@ -138,12 +140,16 @@ func (req updateGroupReq) validate() error {
 	if req.token == "" {
 		return users.ErrUnauthorizedAccess
 	}
+	if req.id == "" {
+		return users.ErrMalformedEntity
+	}
 	if req.Name == "" {
 		return users.ErrMalformedEntity
 	}
 	if len(req.Name) > maxNameSize {
 		return users.ErrMalformedEntity
 	}
```

Comment:
You can `or` these conditions.

Reply:
I meant that you can `or` all of these checks: `if id == "" || name == "" || len(name) > maxNameLen`.

Reply:
I agreed to group this checks by type but for consistency with other services and readability I would keep it like that.
You can check here: https://github.com/mainflux/mainflux/blob/master/things/api/things/http/requests.go